### ISSUE TITLE
학습모델 사용하려면 딥러닝쪽으로 해야될 거 같아.

### ISSUE THREAD

**Issue by chong10** [bana10/capston_2_code#2]
import numpy as np
import tensorflow as tf

# 데이터셋 준비
# X: 이전 프레임의 중심점 위치, Y: 다음 프레임의 중심점 위치
X = np.array([...])  # 이전 프레임 중심점 위치 데이터
Y = np.array([...])  # 다음 프레임 중심점 위치 데이터

# 데이터 정규화
X = (X - np.mean(X)) / np.std(X)
Y = (Y - np.mean(Y)) / np.std(Y)

# 데이터셋 분할
train_ratio = 0.8
train_size = int(len(X) * train_ratio)
train_X, train_Y = X[:train_size], Y[:train_size]
test_X, test_Y = X[train_size:], Y[train_size:]

# RNN 모델 정의
model = tf.keras.Sequential([
    tf.keras.layers.SimpleRNN(64, activation='relu', input_shape=(sequence_length, input_dim)),
    tf.keras.layers.Dense(2)  # 이동 방향 예측을 위한 출력 레이어, 2는 이동 방향 벡터의 차원
])

# 모델 컴파일
model.compile(optimizer='adam', loss='mse')

# 모델 학습
model.fit(train_X, train_Y, epochs=10, batch_size=32)

# 테스트 데이터로 모델 평가
loss = model.evaluate(test_X, test_Y)
print(f'Test loss: {loss}')

# 새로운 입력에 대한 예측
new_input = np.array([...])  # 이전 프레임 중심점 위치 데이터
normalized_input = (new_input - np.mean(X)) / np.std(X)
predicted_output = model.predict(np.expand_dims(normalized_input, axis=0))

큰 틀은 잡아봤는데 데이터셋이 고민이야.
